### DISCUSSION TITLE
Any 'e-books' that are hosted on GitHub?

### DISCUSSION THREAD

**Discussion by AnaisUrlichs** [EddieHubCommunity/support#690] [Q&A]
Heya, I am currently doing a challenge called "100 Days of Kubernetes" for each day, I write done a selection of notes [on my public notion page](https://www.notion.so/researchandstudy/100DaysOfKubernetes-0c1e0de84d654e59bdf89242195abb22). Those notes are obviously not perfect. Thus, I would like to allow others to contribute. 

Does anyone know of examples, in a large number of notes is hosted on GitHub? This would be highly appreciated for inspiration.

Thanks to everyone in advance :slightly_smiling_face:

**Comment by BOLT04** [comment]
Hi @AnaisUrlichs 😃, I don't know any examples that use GitHub to host their notes but while searching I found this repo: https://github.com/darshanime/notes. It doesn't publish the notes as a website, but it's kinda similar to what you wanted.

Your idea sounds great 👍 💯! This might not be the same as notes but _awesome lists_ usually get a lot of traction and contribution, for example, this one on Kubernetes: https://github.com/ramitsurana/awesome-kubernetes.

Hope that helped 😃.

**Reply by anais-codefresh** [reply]
Hey @BOLT04 thank you for your reply, this definitely gives me some inspiration :)

**Comment by eddiejaoude** [comment]
Your notes are great!! 👍 

I host an ebook on GitHub, it is a bit outdated now, but the principles still the same https://github.com/eddiejaoude/book-open-source-tips

- Asciidocs (but can use markdown)
- AsciiDoctor to generate PDF and website
- GitHub Pages to host
- GitHub Action to deploy (mine is currently on TravisCI but it is on my list to migrate over)

**Reply by anais-codefresh** [reply]
Thank you for the suggestions @eddiejaoude :)

**Comment by mikeysan** [comment]
In addition to the ideas above, you could also look into something like [GitHub pages and Jekyll ](https://docs.github.com/en/github/working-with-github-pages/about-github-pages-and-jekyll). This is the same as GitHub pages, but you would be using Jekyll to make the process of creating each page easier; if I understood the documentation correctly. :)

**Reply by anais-codefresh** [reply]
Ah, thank you for your input, this sounds interesting :) will take a look

**Comment by eddiejaoude** [comment]
What did you decide to use @anais-codefresh ?
